src/Workspaces/Core/Portable/Storage/SQLite/Interop/SqlConnection.cs

```diff
@@ -55,7 +54,11 @@ public static SqlConnection Create(IPersistentStorageFaultInjector faultInjector
             // Use SQLITE_OPEN_NOMUTEX to enable multi-thread mode, where multiple connections can be used provided each
             // one is only used from a single thread at a time.
             // see https://sqlite.org/threadsafe.html for more detail
-            var flags = OpenFlags.SQLITE_OPEN_CREATE | OpenFlags.SQLITE_OPEN_READWRITE | OpenFlags.SQLITE_OPEN_NOMUTEX;
+            var flags = OpenFlags.SQLITE_OPEN_CREATE |
+                OpenFlags.SQLITE_OPEN_READWRITE |
+                OpenFlags.SQLITE_OPEN_NOMUTEX |
+                OpenFlags.SQLITE_OPEN_SHAREDCACHE |
+                OpenFlags.SQLITE_OPEN_URI;
             var result = (Result)raw.sqlite3_open_v2(databasePath, out var handle, (int)flags, vfs: null);
```

Comment:
@ericsink can you review this part.  the OPEN_URI is necessary to use the `file:` form for `attach database`.  What's *less* clear is if i need OPEN_SHAREDCACHE.  or if i'll have problems with NO_MUTEX (which we were using before).

Reply:
The flags you've got now are the ones I would think to be correct.